server-code/Projects.js

```diff
@@ -78,3 +89,65 @@ function getProjectsObj(wbsNum) {
     };
     return project;
 }
+
+/** 
+ * Goes through spreadsheet data to get the WBS#s for the work packages associated with the given project and
+ * then returns a 2D array containing that information.
+ * 
+ * @param {String} project – The Work Breakdown Structure # to find data/build a project object for
+ * @return {String[][]} – A 2-D Array representing the table of WBS#s
+ */
+function getProjectWorkPackagesTable(project) {
+    var data = getSheetInfo(MAIN_SHEET_ID_STR, WORK_PACKAGES_STR, DATA_STR);
+    var headers = data[0];
+    var wbsColIdx = findIdx("WBS #", headers);
+    var projWBSNum = project.wbsNum;
+    var projWithoutLastChar = projWBSNum.slice(0, -1);
+    var projectWorkPackagesTable = [["Project WPs"]];
+    for (var rowIdx = 1; rowIdx < data.length; rowIdx++) {
+        var wpWBSNum = data[rowIdx][wbsColIdx];
+        if (wpWBSNum.slice(0, projWithoutLastChar.length) === projWithoutLastChar) {
+            projectWorkPackagesTable.push([wpWBSNum]);
+        }
+        // in this case, you have already discovered the associated WPs, so no unnecessary iterations
+        // compare to len 1 because you'll always have the header list
+        else if (projectWorkPackagesTable.length > 1) {
+            break;
+        }
+    }
+    return projectWorkPackagesTable;
+}
+
+/**
+ * Builds HTML description list from fields in given project.
+ * 
+ * @param {Object} project – The project whose fields to build a description list for
+ * @return {String} – Raw HTML description list built from the given project's fields 
+ */
+function getProjectHtml(project) {
+    var projectWorkPackagesTable = getProjectWorkPackagesTable(project);
+    var html = `<div class="data-frame">
+                    <dl class="row">
+                        <dt class="col-sm-3">Project Name</dt>
+                        <dd class="col-sm-9">` + project.name + `</dd>
+                        <dt class="col-sm-3">WBS #</dt>
+                        <dd class="col-sm-9">` + project.wbsNum + `</dd>
+                        <hr>
+                        <dt class="col-sm-3">Project Lead</dt>
+                        <dd class="col-sm-9">` + project.projectLead + `</dd>
+                        <dt class="col-sm-3">Project Manager</dt>
+                        <dd class="col-sm-9">` + project.projectManager + `</dd>
+                        <hr>
+                        <dt class="col-sm-3">Slide Deck Link</dt>
+                        <dd class="col-sm-9">` + getHTMLLink(project.slideDeckLink, SLIDE_DECK_STR) + `</dd>
+                        <dt class="col-sm-3">BOM Link</dt>
+                        <dd class="col-sm-9">` + getHTMLLink(project.bomLink, BOM_STR)  + `</dd>
+                        <hr>
+                        <dt class="col-sm-3">Project WPs</dt>
+                        <dd class="col-sm-5">` + buildTableHTML(projectWorkPackagesTable)  + `</dd>
```

Comment:
Remember that since you are using backticks, you can use the substitution feature to avoid string concatenation